### ISSUE TITLE
garadar wolf riders (west entrance & east entrance)

### ISSUE THREAD

**Issue by Xadras** [Looking4Group/L4G_Core#875]
Originally reported by: **Bulldoze (Bitbucket: [Legitcow](https://bitbucket.org/Legitcow), GitHub: Unknown)**

----------------------------------------

They are patrolling, but their runspeed is set way too high, they are running way too faster than me on my 100% mount. Also their pathing is poorly scripted, as they stop like after every 3 meter.

----------------------------------------
- Bitbucket: https://bitbucket.org/looking4group_b2tbc/looking4group/issue/879

**Comment by AnonXS** [comment]
https://bitbucket.org/looking4group_b2tbc/looking4group/issues/528/waypoints-wolfreiter-von-garadar

**Comment by Saltgurka** [comment]
The waypoints are fine I think. The formation distance was way too long, so I changed that. The speed was lowered in patch 2.4, so i didn't change that: http://www.wowhead.com/npc=19068/garadar-wolf-rider#comments:id=299027
As for the way they pause at every waypoint, that's just how creature_formations work on our core. I can't do anything about that.

I added a bow to their equip_template and tried to implement the Shoot spell. However every time after combat they refused to start running on their path again, so I've left the spell out for now.
(Also had to use a different shoot spell, because this one doesn't exist in our DB: http://www.wowhead.com/npc=19068/garadar-wolf-rider)

Leaving the issue open, since the spell is still not implemented.

`-- Give Garadar Wolf Riders and Garadar Defenders a bow.
UPDATE `creature_equip_template` SET `equipmodel3`=9060,`equipinfo3`=251789826,`equipslot3`=1  WHERE `entry` = 387;

-- Fix formation distance. It was way too long
DELETE FROM `creature_formations` WHERE `leaderGUID`=68370;
INSERT INTO `creature_formations` (`leaderGUID`,`memberGUID`,`dist`,`angle`,`groupAI`)VALUES
(68370,68370,0,0,2),
(68370,68371,7,0,2);

DELETE FROM `creature_formations` WHERE `leaderGUID`=68368;
INSERT INTO `creature_formations` (`leaderGUID`,`memberGUID`,`dist`,`angle`,`groupAI`)VALUES
(68368,68368,0,0,2),
(68368,68369,7,0,2);
-- -----------------------`
